### PR TITLE
fix(ui) Fix bookmark star not highlighting

### PR DESCRIPTION
**Pull request by markstory**
emotion generates a few different CSS snippets for this component with equivalent specificity. This results in incorrect visual appearance depending on snippet insertion order. Using `!important` overrules the specificity issue. I haven't addressed the missing hover state or wonky colouring of this button but those issues have existed for a long time.

### CODE REVIEW

File: static/app/views/organizationGroupDetails/header.tsx [hunk 1]
```diff
@@ -136,10 +136,6 @@ class GroupHeader extends React.Component<Props, State> {
 
     let className = 'group-detail';
 
-    if (group.isBookmarked) {
-      className += ' isBookmarked';
```
Comment (billyvg) [inline static/app/views/organizationGroupDetails/header.tsx:140]: 👍 doesn't look like this is used in our CSS anywhere.

**Review by billyvg — APPROVED** [review]

File: static/app/views/organizationGroupDetails/header.tsx (hunk 1 above)
Comment (markstory) [reply static/app/views/organizationGroupDetails/header.tsx:140]: I couldn't find any hits either. The other classnames nearby are still used.

File: static/app/views/organizationGroupDetails/actions/index.tsx [hunk 2]
```diff
@@ -319,9 +319,9 @@ const BookmarkButton = styled(ActionButton)<{isActive: boolean}>`
   ${p =>
     p.isActive &&
     `
-    background: ${p.theme.yellow100};
-    color: ${p.theme.yellow300};
-    border-color: ${p.theme.yellow300};
+    background: ${p.theme.yellow100} !important;
+    color: ${p.theme.yellow300} !important;
+    border-color: ${p.theme.yellow300} !important;
     text-shadow: 0 1px 0 rgba(0, 0, 0, 0.15);
```
Comment (priscilawebdev) [inline static/app/views/organizationGroupDetails/actions/index.tsx:325]:
```suggestion
   && {
 background: ${p.theme.yellow100};
 color: ${p.theme.yellow300};
 border-color: ${p.theme.yellow300};
 text-shadow: 0 1px 0 rgba(0, 0, 0, 0.15);
}
```

🤷‍♀️

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**